src/main.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createStore, applyMiddleware } from 'redux';
+import ReactDom from 'react-dom';
+import { Provider } from 'react-redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+
+import App from './component/app/app';
+// import categoryReducer from './reducer/category';
+import reducer from './reducer/main';
+import './styles/main.scss';
+import session from './lib/redux-session';
+
```

Comment:
Import your `./lib/redux-category` file here and add to your applyMiddleware arguments just as your are with `session`. This will allow the console logs to appear with each action and current state of your app.